code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -51,6 +51,9 @@ var/global/list/robot_modules = list(
 	var/list/original_languages = list()
 	var/list/added_networks = list()
 
+	// Station Manifest Stuff
+	var/on_manifest = FALSE
```

Comment:
It would be far better to just make this default to true and only have to be manually put to false.